FreeRTOS-Plus/Demo/coreSNTP_Windows_Simulator/SNTPClientTask.c

```diff
@@ -856,19 +856,19 @@ static void sntpClient_GetTime( SntpTimestamp_t * pCurrentTime )
     xSemaphoreGive( xMutex );
 
     /* Convert UTC time from UNIX timescale to SNTP timestamp format. */
-    ntpSecs = currentTime.secs + SNTP_TIME_AT_UNIX_EPOCH_SECS;
+    ntpSecs = ( uint64_t ) ( currentTime.secs + SNTP_TIME_AT_UNIX_EPOCH_SECS );
```

Comment:
```suggestion
    ntpSecs = ( uint64_t ) ( currentTime.secs ) + ( uint64_t ) ( SNTP_TIME_AT_UNIX_EPOCH_SECS );
```
If both `currentTime.secs` and `SNTP_TIME_AT_UNIX_EPOCH_SECS` are of type `uint32_t`, the following condition `if( ntpSecs > UINT32_MAX )` will never be true.